spec/controllers/external_users/advocates/claims_controller_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 RSpec.describe ExternalUsers::Advocates::ClaimsController do
   let!(:advocate) { create(:external_user, :advocate) }
+  let!(:date) { Time.zone.today - 1.year }
```

Comment:
This doesn't need a `!` on the `let`. `let` will only evaluate when the variable is first used while `let!` ensures that the value is always calculated.